### DISCUSSION TITLE
Same response time in 1 core and 8 core aws gpu

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#158] [Archived]
<USER>>>> halder.nayan35<br/> <DATE> [March 10, 2020, 8:14am]<br/> <br/> Hi all, *slash* <br/> I am running mozilla TTS on 8 core AWS GPU and the response time is same<br/> as 1 core.<br/> <br/> i first run the 1) export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 *slash* <br/> 2) source tts-venv/bin/activate *slash* <br/> 3) python -m TTS.server.server<br/> <br/> Am i missing any configuration setting.<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/same-response-time-in-1-core-and-8-core-aws-gpu]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> dkreutz<br/> <DATE> [March 10, 2020, 9:19pm]<br/> <br/> No, to my understanding inference/synthesizer can not use multiple GPUs<br/> as there is no batch processing possible like when training a model.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> nmstoker<br/> <DATE> [March 10, 2020, 11:01pm]<br/> <br/> Yes, using one GPU is consistent with Eren's response here:<br/> <https://github.com/mozilla/TTS/issues/363><br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> halder.nayan35<br/> <DATE> [March 12, 2020, 5:03pm]<br/> <br/> Thanks for the reply. I understood 8 core gpu will not improve response<br/> time. Is there a way out to improve TTS response time within 1 core AWS<br/> gpu. My current TTS response time 4-5 sec. I want to bring down to this<br/> below 1-2 sec. Is it doable in 1 core gpu by some configuration changes<br/> or settings.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [March 12, 2020, 11:37pm]<br/> <br/> number of GPUs does not do any good for a single utterance response<br/> time. You can only increase the number of utterances processed in<br/> parallel.<br/> <br/> [Archived Post]<br/>

**Reply by chikiuso** [reply]
Hi @JRMeyer , may I ask how to increase the number of utterances processed ? thanks.
